src/iceberg_explorer/catalog/service.py

```
@@ -0,0 +1,251 @@
+            Dictionary containing table metadata including:
+            - location: Table location (URI)
+            - snapshot_id: Current snapshot ID
+            - partition_spec: Partition specification
+            - snapshots: List of snapshot dictionaries with sequence_number,
+              snapshot_id, timestamp_ms, and manifest_list
+
+        Raises:
+            RuntimeError: If the catalog type is not supported.
+            NoSuchTableError: If a table does not exist.
+        """
+        table_identifier = tuple(namespace.split(".") + [table_name])
```

Ruff (0.14.13)

Consider `[*namespace.split("."), table_name]` instead of concatenation

Replace with `[*namespace.split("."), table_name]`

(RUF005)